.github/workflows/docker_publish.yml

```diff
@@ -261,9 +263,10 @@ jobs:
     name: Installing and Testing DAGMC
     steps:
       - name: Checkout repository
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
         with:
           submodules: recursive
+          set-safe-directory: "/__w/DAGMC/DAGMC"
```

Comment:
For my education, what does the syntax here mean? Is this the default location of repo checkouts using the GHA tool?

I think we want to be explicit about this safe directory as you've done here @shimwell, but I'll paste this option here too from the relevant change in the GHA checkout action
 PR in case it's useful in the future. 

https://github.com/actions/checkout/pull/770#issuecomment-1104448853

Reply:
this was based on the git error message, that identified the folder directory